packages/perspective-viewer-d3fc/src/js/d3fc/extent/extent.js

```diff
@@ -0,0 +1,137 @@
+/******************************************************************************
```

Comment:
Filename `linearExtent.js` to match the component name? Just in case we need to add more custom extents...